pulumi/aws/anthos/__main__.py

```diff
@@ -38,12 +48,26 @@ def pulumi_ingress_project_name():
 ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
 lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
 
-# Spin up a new instance of the kuard demo app by creating: a Deployment resource,
-# a Service resource, and an Ingress resource that are all defined within a single
-# Python class named App.
-app = App('kuard',
-          AppArgs(image='gcr.io/kuar-demo/kuard-amd64:blue',
-                  host=lb_ingress_hostname),
-          opts=pulumi.ResourceOptions(provider=k8s_provider))
+ns = k8s.core.v1.Namespace(resource_name='boa',
+                           metadata={'name': 'boa'},
+                           opts=pulumi.ResourceOptions(provider=k8s_provider))
+
```

Comment:
What do you think about adding a function like:

```python
def use_ingress_hostname(obj):
    if 'spec' in obj and 'rules' in obj['spec']:
        for item in obj['spec']['rules']:
            item['host'] = lb_ingress_hostname
```

And adding it to the ingress manifest transformations on line 71? This would allow us to access the Bank of Anthos app directly at the ELB domain. Alternatively, we could mount it on a subdirectory to the domain. If we do that we could then direct to other Kubernetes apps like Graphana (although in the graphana / prometheus case we may want a different cluster) or the [k8s dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/).

Reply:
I went a different way and just added in the lb_ingress_hostname as the hostname for the deployment. Since the ingress is now generated entirely in pulumi code we can grab an alternative hostname from the config, but if not found we just use the LB.